src/ucp/proto/proto_multi.inl

```diff
@@ -184,7 +194,8 @@ ucp_proto_multi_bcopy_progress(ucp_request_t *req,
     return ucp_proto_multi_progress(req, mpriv, send_func, comp_func, UINT_MAX);
 }
 
-static UCS_F_ALWAYS_INLINE ucs_status_t ucp_proto_multi_zcopy_progress(
+static UCS_F_ALWAYS_INLINE ucs_status_t
+ucp_proto_multi_zcopy_progress_custom_lane(
```

Comment:
We need to distinguish two functions (`ucp_proto_multi_zcopy_progress_custom_lane` and `ucp_proto_multi_zcopy_progress`): first for rndv/put|get/zcopy protocols and second for other.
The first function without lane advance (since it advances in send callback), the second with lane advance.